pkg/secretless/plugin/connector/README.md

```diff
@@ -0,0 +1,267 @@
+# Secretless Connector Plugins
+
+For general info on Secretless plugins, please refer to the [plugin README](../README.md).
+
+Secretless has built-in support for PostgreSQL, MySQL, APIs that use basic HTTP authentication, and many other target services.
+
+But what if you want to use Secretless with a target service it doesn't support out of the box?
+
+Service connector plugins let you extend Secretless to support any target service. If you know Go and understand
+your target service's authentication protocol, you can write a connector for it. It's easy and this guide walks you through it.
+
+The Secretless team is continually adding support for new databases and services, but we love and encourage outside
+contributions as well. If you write a connector plugin you'd like to share with the community, please
+[let us know](https://secretless.io/community.html) or consider sending us a PR!
+
+## Technical Overview
+
+As described in the [plugin README](../README.md), Secretless uses [Go plugins](https://golang.org/pkg/plugin/)
+to produce shared object library files (with `.so` extensions) instead of a normal executable. The `.so` library files can
+be loaded at runtime to expand the number of target services your Secretless instance supports.
+
+Technically, a Secretless service connector plugin is just a Go shared library
```

Comment:
I went with 
> Technically, a Secretless service connector plugin is a Go shared library file that implements two functions, either:

because it seemed clearer to me - please lmk if you object